beacon-chain/forkchoice/protoarray/nodes.go

```diff
@@ -47,8 +47,7 @@ func (s *Store) head(ctx context.Context, justifiedRoot [32]byte) ([32]byte, err
 	// Update metrics.
 	if bestNode.root != lastHeadRoot {
 		headChangesCount.Inc()
-		headSlotNumber.Set(float64(bestNode.Slot))
-		nodeCount.Set(float64(len(s.nodes)))
+		headSlotNumber.Set(float64(bestNode.slot))
```

Comment:
Seems like buildkite fails because this needs to be `bestNode.Slot`

Reply:
Thanks!